apps/src/codebridge/FilePreview/index.tsx

```diff
@@ -33,7 +33,7 @@ const getPreviewComponent = (
   }
 };
 
-export const PreviewContainer = () => {
+export const FilePreview = () => {
```

Comment:
I think there was an intent of maybe previewing other types of files, I'm not sure. There is already an `HTMLPreview` file in the same folder that is the only type of preview right now. Because the file was written pretty generically I might leave it as-is for now, I could see us refactoring in the future to just make this an html preview and remove the generic code in there.r